app/Models/Airport.php

```diff
@@ -54,7 +74,7 @@ public function getRows(): array
                 'iata' => strtoupper($details['iata']),
                 'name' => $details['name'],
                 'city' => $details['city'],
-                'state' => $details['state'],
+                'state' => str_replace('-', ' ', $details['state']),
```

Comment:
For some reason, this dataset has dashes where there should be spaces. This fixes that.